meson.build

```diff
@@ -526,15 +533,37 @@ crypto = declare_dependency(
     include_directories: ssl_includes,
 )
 
-if ssl_link_args != [] and ssl_includes != []
+if wolfssl.found() or ssl_link_args != []
     have_ssl = true
     cdata.set('HAVE_LIBCRYPTO', 1)
-    cdata.set('OPENSSL_DHX', 1)
     cdata.set('UAM_DHX', 1)
+endif
+
+ssl_deps = []
+ssl_type = ''
+
+#if have_embedded_ssl and wolfssl.found()
+#    error('embedded SSL cannot be compiled when WolfSSL is installed')
```

Comment:
Do you plan to use this commented out code at some point?

Reply:
No!